spec/furniture/marketplace/delivery_areas_controller_request_spec.rb

```diff
@@ -12,10 +12,18 @@
   let(:marketplace) { create(:marketplace) }
   let(:person) { create(:membership, space: space).member }
 
+  describe "#index" do
+    let(:perform_request) { get polymorphic_path(marketplace.location(child: :delivery_areas)) }
+
+    it { is_expected.to be_ok }
+    specify { perform_request && assert_select("a", "Add Delivery Area") }
+  end
+
   describe "#new" do
     let(:perform_request) { get polymorphic_path(marketplace.location(:new, child: :delivery_area)) }
 
     it { is_expected.to be_ok }
+    specify { perform_request && assert_select("form input", 3) && assert_select("#delivery_area_label") && assert_select("#delivery_area_price") }
```

Comment:
If this is too long, I can break into separate `specify` blocks.

Reply:
I think this is fine for now. I tend to prefer asserting that the request renders a particular partial or component rather than specifying the view details in the request specs; but until we get good at writing component specs I think this is a fine option!